libs/api/domains/license-service/src/lib/client/driving-license-client/drivingLicenseService.api.ts

```diff
@@ -158,9 +163,51 @@ export class GenericDrivingLicenseApi
     return licenses
   }
 
+  private async getCachedGetPkPassToken(): Promise<string | null> {
+    if (!this.cacheManager) {
+      return null
+    }
+
+    const cached = await this.cacheManager?.get(CACHE_KEY)
+
+    if (cached && typeof cached === 'string' && cached.length > 0) {
+      return cached as string
```

Comment:
```suggestion
      return cached
```

we've already establised that `typeof cached === 'string'` in the line above